infrastructures/infrastructure-common/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/util/LinuxInitScriptGenerator.java

```diff
@@ -91,6 +108,42 @@ private String generateNodeStartCommand(String instanceId, String rmUrlToUse, St
         return "nohup " + javaCommand + javaProperties + "  &";
     }
 
+    public String generateDefaultIaasConnectorURL(String DefaultRMHostname) {
+        if (nsConfig == null) {
+            try {
+                // If the configuration manager is not loaded, I load it with the NodeSource properties file
+                nsConfig = NSProperties.loadConfig();
+            } catch (ConfigurationException e) {
```

Comment:
I suggest to remove the try catch block and use throws if possible.

Reply:
I'm not a huge fan of this solution: launching exception at this point will have important consequences on infra class working with it: E.g. GCEInfrastructure cloud won't be able to instanciate although the user have supplied a custom IaaS-connector string. I prefer to bet on resilience.